packages/common/src/fields/Image.ts

```diff
@@ -0,0 +1,55 @@
+import Field from './Field'
+
+type AllowedDisks = "public" | "private"
+
+export class Image extends Field {
+    public selectedDisk: AllowedDisks = "public"
+
+    public component = 'FileField'
+
+    /**
+     * When a new file is made, 
+     * we'll allow it to be downloaded
+     *
+     */
+    public constructor(name: string, databaseField?: string) {
+        super(name, databaseField)
+
+        this.attributes = {
+            maxSize: 20000
```

Comment:
This max size should be on the File class instead. The File class should also have `allowedMimeTypes` property, to know which files are allowed. By default `*` should be allowed. Then the user can define other types such as `png`, `.csv` etc